YIF_Backend/Controllers/UsersController.cs

```diff
@@ -206,10 +206,13 @@ public async Task<IActionResult> Restore([FromBody] RestoreApiModel model)
         [ProducesResponseType(typeof(DescriptionResponseApiModel), 400)]
         [ProducesResponseType(typeof(ErrorDetails), 500)]
         [HttpPut("ChangePassword")]
+        [Authorize]
         public async Task<IActionResult> ChangePassword([FromBody] ChangePasswordApiModel model)
         {
             if (!ModelState.IsValid) 
```

Comment:
you don't need such validation because you already have fluent validation and validation at service